app/main/views/broadcast.py

```diff
@@ -135,22 +135,46 @@ def new_broadcast(service_id):
 def write_new_broadcast(service_id):
```

Comment:
Does it make sense to change the name of this function now?

write_new_broadcast could be a confusing name since it doesn't always create a new broadcast anymore

Reply:
I'd argue it should still be write new broadcast because it's not necessarily intended as being able to have create/edit capacity - the idea is just that whatever is entered into the form isn't lost if coming back to it from later on in the flow.